docs/hardware.rst

```diff
@@ -10,17 +10,17 @@ Required Hardware
 -----------------
 
 Servers
-~~~~~~~
+^^^^^^^
 
 These are the core components of a SecureDrop instance.
 
--  *Application Server*: 1 physical server to run the SecureDrop web
-   services.
--  *Monitor Server*: 1 physical server which monitors activity on the
-   *Application Server* and sends email notifications to an
-   administrator.
--  **Network Firewall**: 1 physical computer that is used as a dedicated
-   firewall for the SecureDrop servers.
+* *Application Server*: 1 physical server to run the SecureDrop web services.
+
+* *Monitor Server*: 1 physical server which monitors activity on the
+  *Application Server* and sends email notifications to an administrator.
+
+* **Network Firewall**: 1 physical computer that is used as a dedicated firewall
```

Comment:
Thanks -- this was already there, and something I meant to ask about. Will change.